src/plugins/data/common/search/search_source/search_source.ts

```diff
@@ -917,7 +917,7 @@ export class SearchSource {
   /**
    * serializes search source fields (which can later be passed to {@link ISearchStartSearchSource})
    */
-  public getSerializedFields(recurse = false): SerializedSearchSourceFields {
+  public getSerializedFields(recurse = false, includeFields = true): SerializedSearchSourceFields {
```

Comment:
Nit: Can we make this an object so the arguments are more explicit?
```suggestion
  public getSerializedFields({
    recurse = false,
    includeFields = true,
  }): SerializedSearchSourceFields {
```